### PR TITLE
Small documentation fixes

### PR DESCRIPTION
**Pull request by shagunsodhani**
## Motivation

Small documentation fixes

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

NA

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: website/docs/terminology.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ The config for the application. It is a dictionary of [Config Nodes](#config-nod
 ## Misc
 ### Config Search Path
 The [Config Search Path](advanced/search_path.md) is a list of paths that are searched in order to find configs. It is similar to
-the Python PYTHONPATH.
+the Python [PYTHONPATH](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH).
```
Comment (omry) [inline website/docs/terminology.md:87]: not that similar, you can't override it through environment. dunno if I want that link here.
It is conceptually similar.

File: website/docs/tutorials/basic/your_first_app/3_using_config.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ def my_app(cfg: DictConfig):
     assert cfg["node"]["loompa"] == 10    # dictionary style access
 
     assert cfg.node.zippity == 10         # Value interpolation
-    assert type(cfg.node.zippity) == int  # Value interpolation type
+    assert isinstance(cfg.node.zippity, int)  # Value interpolation type
```
Comment (omry) [inline website/docs/tutorials/basic/your_first_app/3_using_config.md:24]: this is backed by an actual example in the examples. update it too.

File: website/docs/terminology.md (hunk 1 above)
Comment (shagunsodhani) [reply website/docs/terminology.md:87]: Yeah I think a better analogy is sys.path

Comment (omry) [reply website/docs/terminology.md:87]: sys.path is am implementation detail of how this is done in Python.
this is all modeled after unix PATH.
I think in general it's okay to say it's similar to PYTHONPATH with providing more details.

Comment (shagunsodhani) [reply website/docs/terminology.md:87]: So any change needed ?

Comment (omry) [reply website/docs/terminology.md:87]: we can keep this change.

File: website/docs/tutorials/basic/your_first_app/3_using_config.md (hunk 2 above)
Comment (omry) [reply website/docs/tutorials/basic/your_first_app/3_using_config.md:24]: this one needs an update. (the example is in examples).

**Comment by omry** [comment]
1. Thanks for going over the entire tutorial!
2. You have some failing tests :)

**Comment by omry** [comment]
ah, this is related to an OmegaConf upgrade.
I am landing an update to Hydra soon.

**Comment by omry** [comment]
#790

**Comment by omry** [comment]
merged, please rebase (and upgrade omegaconf locally).

**Pull request merged**